README.md

```diff
@@ -28,7 +28,7 @@ powershell "[Net.ServicePointManager]::SecurityProtocol = 'tls12'; irm https://d
 
 For scripts, please see [Detect Scripts](https://github.com/synopsys-sig/synopsys-detect-scripts).
 
-For AirGap, please use our [Artifactory](https://sig-repo.synopsys.com/webapp/#/artifacts/browse/tree/General/bds-integrations-release/com/synopsys/integration/synopsys-detect).
+For AirGap, please use our [Artifactory](https://sig-repo.synopsys.com/webapp/#/artifacts/browse/tree/General/bds-integrations-release/com/blackduck/integration/synopsys-detect).
```

Comment:
If we are doing these changes, might as well change to repo.blackduck.com and remove synopsys from the jar name.

Reply:
Are these changes supposed to be done by Chris?

Reply:
Done, and yes. I have requested his review.

Reply:
FYI I have the much related https://github.com/blackducksoftware/synopsys-detect/pull/1243 open but was holding it until we have final confirmation on tht GitHub paths for Jenkins.